### DISCUSSION TITLE
Mobile UI / PWA

### DISCUSSION THREAD

**Discussion by wehbs** [8ff/firescrew#8] [Ideas]
Wanted to start a thread to brainstorm new ideas or discuss any planned mobile UI enhancements.

**Comment by chiefMarlin** [comment]
Lets do this!

**Reply by chiefMarlin** [reply]
Im the meantime, i am working on improving the object detection and making some performance improvents, however once this is finished UI will be the next focus.
So please feel free to pitch in whatever ideas you got

**Comment by wehbs** [comment]
I think maybe a quick fast follow could be just adding some media queries to adjust the ui when the viewport is on mobile.

**Reply by chiefMarlin** [reply]
Hey, that's a solid suggestion!
Would you be interested in implementing something like this and creating a PR?

**Comment by wehbs** [comment]
Yes, for sure!

**Reply by chiefMarlin** [reply]
That would be awesome! Thanks!

**Reply by chiefMarlin** [reply]
fyi, the web ui static files are located in pkg/firescrewServe/static

**Comment by chiefMarlin** [comment]
Thanks for the PR, main page looks much more pleasant on mobile now!
We may be able to do some further modifications to the modal, but ofcourse mobile experience is always somewhat limited
